### PR TITLE
Add Base Container testing

### PR DESCRIPTION
**Pull request by griffinandrew**
The test short-master contains all the appropriate image, build, and UI tests compressed into one. The build step is responsible for building the image, pushing it to the registry of quay.io, and recording the time of the build. All other jobs wait for this to complete before beginning. 

1. health-check 
This test pulls the latest built image and, runs it, after some time executes the health status to check if the container is up and running

2. image-version-check 
This test pulls the latest built image and, runs it, then queries the image json to ensure the proper image and tag are used

3. package-vers-test
This test checks what version of packages are installed within the running container and then passes that to a Python script that is then used to compare the current list with a stable list and find any inconsistencies

4. checksum 
Checksum verifies that the checksum of the container image is correct 

5. size and time
This test uses the build time recorded in the build to display that time and also queries the image to find its size and show that as well

6. UI Test
This test infrastructure uses selenium to run functionality and visual checks on jupyter notebook running in the image 

NOTE: in order to push / pull to the appropriate quay.io repo you must update registry_user and registry_password to the correct  corresponding variables

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -72,92 +65,76 @@ build: DARGS ?= --build-arg FROM_REG=$(BASE_REG) \
...
-show-run-beta: ARGS ?=
-show-run-beta: DARGS ?= -u $(OPE_UID):$(OPE_GID) -v "${HOST_DIR}":"${MOUNT_DIR}" -v "${SSH_AUTH_SOCK}":"${SSH_AUTH_SOCK}" -v "${SSH_AUTH_SOCK}":"${SSH_AUTH_SOCK}" -e SSH_AUTH_SOCK=${SSH_AUTH_SOCK} -p ${SSH_PORT}:22
-show-run-beta: PORT ?= 8888
-show-run-beta: ## start published version with jupyter lab interface
-	@-echo "docker run -it --rm -p $(PORT):$(PORT) $(DARGS) $(OPE_REGISTRY)$(OPE_IMAGE)$(OPE_TAG) $(ARGS)""
-
-show-tag: ## show current tag
-	@-echo $(OPE_REGISTRY)$(OPE_IMAGE)$(OPE_BETA_TAG)
-
+	docker run -it --rm -p $(PORT):$(PORT) $(DARGS) $(OPE_BOOK_REG)$(OPE_BOOK_IMAGE)$(OPE_PUBLIC_TAG) $(ARGS)
```
Comment (DanNiESh) [inline Makefile]: Can you add an empty new line at the end of file?

File: base/checksum [hunk 2]
```diff
@@ -0,0 +1 @@
+fb2cc759cef3e1a8dd909c66373991a5f028f0bc6b2fcb76bdc7a5b04683b6a5
```
Comment (DanNiESh) [inline base/checksum]: no new line

File: base/customization_name [hunk 3]
```diff
@@ -0,0 +1 @@
+base-ope-image
```
Comment (DanNiESh) [inline base/customization_name]: same as above and below

**Review by DanNiESh — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (griffinandrew) [reply Makefile]: sure, do u mean a physical new line or \n?

**Pull request closed without merging**